bridge/src/main/java/io/stargate/bridge/service/SchemaHandler.java

```diff
@@ -69,7 +69,8 @@ public static void describeKeyspace(
 
   static CqlKeyspaceDescribe buildKeyspaceDescription(Keyspace keyspace) throws StatusException {
 
-    CqlKeyspaceDescribe.Builder describeResultBuilder = CqlKeyspaceDescribe.newBuilder();
+    CqlKeyspaceDescribe.Builder describeResultBuilder =
+        CqlKeyspaceDescribe.newBuilder().setHash(keyspace.hashCode());
```

Comment:
Using the Java hashcode for a quick first prototype.
If we estimate that the risk of collision is too high, it should be pretty easy to implement another algorithm.

Reply:
This has to be good enough.. Has to be!

Reply:
For initial prototype yes, but I would suggest that instead of Java String.hash() which is not really very high quality, we'd use same thing as Cassandra: Murmur3 hash:

https://docs.datastax.com/en/cassandra-oss/3.x/cassandra/architecture/archPartitionerM3P.html

Murmur algorithm is not cryptographic but it is very fast, has good distribution. I think you can also make it produce 64-bit hash which I think would be better than 32-bit for our case.

I know this is bit of an internal implementation detail but since it will be communicated across Bridge we need to decide on it relatively early.

Reply:
As per our discussion, my point was off: client does NOT need to (re)compute hash at all. So it's a server-side implementation detail, except for type used (which needs to be agreed on).

Given this, using Java classes `hashCode()` is reasonable initial implementation. Can be changed if need be, without clients having to care.